diff --git a/pyproject.toml b/pyproject.toml
index 95fd6b2..2aaf531 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["poetry-core==2.1.3", "taskipy==1.14.1"]
+requires = []
 build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
@@ -28,7 +28,7 @@ build123d = ">=0.9,<0.10"
 # Misc
 pygltflib = "^1.16.2"
 pillow = ">=10.2,<12.0"
-poetry-core = "==2.2.1"
+poetry-core = ">=2.1.1"
 
 [tool.poetry.group.dev.dependencies]
 taskipy = "^1.14.1"
