diff --git a/pywrap/bindgen/translation_unit.py b/pywrap/bindgen/translation_unit.py
index 3f5e045..a8ab85f 100755
--- a/pywrap/bindgen/translation_unit.py
+++ b/pywrap/bindgen/translation_unit.py
@@ -12,6 +12,7 @@ def parse_tu(path,
              prefix=None,
              platform_includes=[],
              args=['-x', 'c++', '-std=c++17','-D__CODE_GENERATOR__',
+                   '-Wno-macro-redefined',
                    '-Wno-deprecated-declarations'],
              parsing_header = '',
              platform_parsing_header = ''):
