diff --git a/pywrap/bindgen/CMakeLists.j2 b/pywrap/bindgen/CMakeLists.j2
index a9ae7d7..aa7ff9f 100755
--- a/pywrap/bindgen/CMakeLists.j2
+++ b/pywrap/bindgen/CMakeLists.j2
@@ -12,7 +12,7 @@ if(WIN32)
   string(STRIP ${PYTHON_VER} PYTHON_VER)
   set( PYTHON_LIBRARY $ENV{CONDA_PREFIX}/libs/python${PYTHON_VER}.lib )
 else()
-  set( PYTHON_EXECUTABLE "$ENV{CONDA_PREFIX}/bin/python" )
+  set( PYTHON_EXECUTABLE "@python@/bin/python" )
 endif()
 
 find_package( pybind11 REQUIRED )
